src/backend/parser/parse_utilcmd.c

```diff
@@ -2031,7 +2031,7 @@ transformDistributedBy(CreateStmtContext *cxt,
 			 * is an inherited table, set the distribution based on the
 			 * parent (or one of the parents)
 			 */
-			if (distrkeys == NIL && parentPolicy->nattrs >= 0)
+			if (distrkeys == NIL && parentPolicy && parentPolicy->nattrs >= 0)
```

Comment:
```suggestion
			if (distrkeys == NIL && parentPolicy != NULL && parentPolicy->nattrs >= 0)
```

Reply:
Is there code style, where there is this requirement?

Reply:
> Is there code style

[yes](https://github.com/arenadata/gpdb/pull/700/files#diff-5bd59ecc8991bacaefd56f7fe986287b8d664e62566eb3588c3845d7625cacf1R2018)

Reply:
I said about rules, not examples in the code.